<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AircraftDialog</class>
 <widget class="QDialog" name="AircraftDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QCheckBox" name="stowedBox">
      <property name="text">
       <string>Stowed</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QSpinBox" name="fuelBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Type of Aircraft</string>
      </property>
      <property name="buddy">
       <cstring>typeBox</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Serial Number</string>
      </property>
      <property name="buddy">
       <cstring>serialInput</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="squadronBox"/>
    </item>
    <item row="3" column="2">
     <widget class="QSlider" name="fuelSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Fuel Level</string>
      </property>
      <property name="buddy">
       <cstring>fuelSlider</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="serialInput"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Squadron</string>
      </property>
      <property name="buddy">
       <cstring>squadronBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="typeBox"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Initial Location</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QComboBox" name="initialLocation">
      <item>
       <property name="text">
        <string>Flight Deck</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hanger</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>On Mission</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>In Transit</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>serialInput</tabstop>
  <tabstop>typeBox</tabstop>
  <tabstop>squadronBox</tabstop>
  <tabstop>fuelSlider</tabstop>
  <tabstop>stowedBox</tabstop>
  <tabstop>fuelBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AircraftDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AircraftDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fuelSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fuelBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fuelBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fuelSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
